private/bufnew/bufconfig/buf_work_yaml_file.go

```diff
@@ -129,13 +129,13 @@ type bufWorkYAMLFile struct {
 }
 
 func newBufWorkYAMLFile(fileVersion FileVersion, dirPaths []string) (*bufWorkYAMLFile, error) {
-	if err := validateBufWorkYAMLDirPaths(dirPaths); err != nil {
+	sortedNormalDirPaths, err := validateBufWorkYAMLDirPaths(dirPaths)
```

Comment:
We've used `normalized` throughout rest of codebase, not `normal`, update

Reply:
fixed in [f2481c7](https://github.com/bufbuild/buf/pull/2613/commits/f2481c785d23efba16fe8534c34d238e3f9256e8)